src/app/components/pages/movies.jsx

```diff
@@ -1,7 +1,13 @@
-import React from 'react';
+import React from "react";
+import MovieListContinue from "../../../shared/MovieList";
+import MovieListPopular from "../../../shared/MovieList";
 
+const Movies = () => {
```

Comment:
Movies jak sama nazwa wskazuje ma mapowac filmy a nie być kontenerem. 
Ta funkcja wyglada na Dashboard

Reply:
nie bardzo rozumiem jak to ma wygladac, w movieList mapuje po api i wyciagam kafelki filmow a w Movies je przekazuje. mam tu znowu uzyc hooka useMovies?